src/com/trollworks/gcs/spell/SpellTechnique.java

```diff
@@ -0,0 +1,214 @@
+        return level;
+    }
+
+    @Override
+    public boolean isEquivalentTo(Object obj) {
+        if (obj == this)
+            return true;
+        if (obj instanceof SpellTechnique) {
+            SpellTechnique that = (SpellTechnique) obj;
+            if (mSpellPrerequisiteCount != that.mSpellPrerequisiteCount)
+                return false;
+            return super.isEquivalentTo(obj);
+        }
+        return false;
+    }
+
+    @Override
+    public String getLocalizedName() {
+        return I18n.Text("Spell (Ritual Magic)");
+    }
+
+    @Override
+    public String getXMLTagName() {
+        return TAG_SPELL_TECHNIQUE;
+    }
+
+    @Override
+    public int getXMLTagVersion() {
+        return CURRENT_VERSION;
+    }
+
+    @Override
+    public String getRowType() {
+        return I18n.Text("Spell (Ritual Magic)");
+    }
+
+    @Override
+    protected void prepareForLoad(LoadState state) {
+        super.prepareForLoad(state);
+    }
+
+    @Override
+    public RowEditor<? extends ListRow> createEditor() {
+        return new SpellTechniqueEditor(this);
+    }
+
+    /**
+     * @param builder The {@link StringBuilder} to append this technique's satisfied/unsatisfied
+     *                description to. May be {@code null}.
+     * @param prefix  The prefix to add to each line appended to the builder.
+     * @return {@code true} if this technique has its default satisfied.
+     */
+    public boolean satisfied(StringBuilder builder, String prefix) {
+        if(getCollege() == null || getCollege().isBlank()) {
+            if (builder != null) {
+                String format = I18n.Text("{0}Must be assigned to a college\n");
+                builder.append(MessageFormat.format(format, prefix));
+            }
+            return false; // Do not check further requirements
+        }
+
+        boolean result = true;
+        String  skillName = I18n.Text("College");
+        String  skillSpec = getCollege();
+        Skill   skill     = getCharacter().getBestSkillNamed(skillName, skillSpec, false, new HashSet<>());
+
+        if (skill == null) {
+            if (builder != null) {
+                String format = I18n.Text("{0}Requires a skill named {1} ({2})\n");
+                builder.append(MessageFormat.format(format, prefix, skillName, skillSpec));
+            }
+            result = false;
+        }
+        return result;
+    }
+
+    public SkillDefault getDefault() {
+        return new SkillDefault(SkillDefaultType.Skill, I18n.Text("College"), getCollege(), -mSpellPrerequisiteCount);
+    }
+
+    public int getSpellPrerequisiteCount() {
+        return mSpellPrerequisiteCount;
+    }
+
+    public boolean setSpellPrerequisiteCount(int spellPrerequisiteCount) {
+        if (mSpellPrerequisiteCount != spellPrerequisiteCount) {
+            mSpellPrerequisiteCount = spellPrerequisiteCount;
+            return true;
+        }
+        return false;
+    }
```

Comment:
Seems like this should really be a calculated value, based on actual prereqs.

Reply:
Thaumatology 72-73 talks about a few issues relative to determining prerequisite count

- How to handle non-spell prerequisites. Options include satisfying them, ignoring them, or making them add a penalty of 1 per 10 full points of their cost.
- Double-counting of prerequisites. If spell D depends on spells B and C, and both depend on spell A (diamond pattern), is the prereq count 3 or 4? (luckily the book suggests 3 but does not force it)
- Some spells like Enchant have crazy prereqs (10 other spells) so the manual says the GM might apply a custom penalty to avoid discouraging players from using it.

So I thought it was best to just store the penalty and let the user modify it as he likes.
If this PR is accepted I was thinking of making a new spell library (or add to the existing one) with the RM variant of all the spells in Magic, with a default prereq count based on the table in Thaumatology 261-267.

What do you think?

Reply:
OK, sounds reasonable to have it just be a input field, then.

As for a new spell list, yes, I think that's best. Probably best to create a new libraries with a " (Ritual Magic)" suffix, i.e. "Magic (Ritual Magic)", etc.